backend/src/data/models/interview/interview.model.ts

```diff
@@ -9,7 +9,13 @@ import { User } from '../models';
 class Interview extends Abstract {
   public 'interviewDate': string;
```

Comment:
If not nullable constraint was dropped
```suggestion
  public 'interviewDate': string | null;
```